src/client/components/pages/entities/author-table.js

```diff
@@ -30,8 +30,8 @@ function AuthorTableRow({author, showAddedAtColumn, showCheckboxes, selectedEnti
 	const name = getEntityLabel(author);
 	const disambiguation = getEntityDisambiguation(author);
 	const number = author.number || '?';
-	const authorType = author.authorType ? author.authorType.label : '?';
-	const gender = author.gender ? author.gender.name : '?';
+	const authorType = author.authorType || author.authorType?.label || author?.type || '?';
+	const gender = author.genderId === 1 ? "Male" : author.genderId ? "Female" : "?";
```

Comment:
As I mentioned on IRC, the right way to do this is to use the [`loadGenders` middleware](https://github.com/metabrainz/bookbrainz-site/blob/816f687b34e2653fbf27eda1488f99b195c73cc0/src/server/helpers/middleware.ts#L69) on the server side (in the [series get endpoint](https://github.com/metabrainz/bookbrainz-site/blob/0e6807c9b6/src/server/routes/entity/series.js#L138)) to load  the genders from the database (which also attaches them to the response 'locals' props).
Then you can pass the genders to the page props (see [example here](https://github.com/metabrainz/bookbrainz-site/blob/0e6807c9b6151dc166e201a3aa2d6425db2902b9/src/server/routes/register.js#L68-L70)), and use that to get the name associated to the `author.genderId`:

something along the lines of `genderOptions.find((option) => option.id === author.genderId).name`